components/Composition/FColorSelect.jsx

```diff
@@ -0,0 +1,42 @@
+import { StyleSheet, View, TouchableOpacity } from 'react-native';
+import React from 'react';
+import sizes from 'themes/sizes';
+import colors from 'themes/colors';
+import opacities from 'themes/opacities';
+
+export const Circle = ({
+  color, value, setValue, size,
+}) => (
+  <TouchableOpacity
+    onPress={setValue}
+  >
+    <View
+      backgroundColor={color}
+      width={size}
+      height={size}
+      style={[styles.circle, value ? styles.pressed : styles.default]}
+    />
+  </TouchableOpacity>
+);
+
+const styles = StyleSheet.create({
+  circle: {
+    borderRadius: sizes.RADIUS_50,
```

Comment:
to wywalić do stylu do view (dlatego że potrzebujesz uzależnić od size) i skorzystać z funkcji getHalfBorderRadius w folderze utils (na samym dole projektu nad envem). Robimy to po to żeby bez względu na to jak wielkie bedzie kółko to i tak zawsze bedzie okrągłe